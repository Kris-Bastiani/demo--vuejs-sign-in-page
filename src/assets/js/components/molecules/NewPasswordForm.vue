<template lang="pug">
	form
		input-field(label='New password:', placeholder='Enter new password', type='password')

		input-field.mrg_b__lrg(label='Confirm password:', placeholder='Confirm new password', type='password')

		input(type='submit', value='Save')
</template>

<script>
import InputField from '../atoms/InputField.vue';

export default {
	components: { InputField },
};
</script>
