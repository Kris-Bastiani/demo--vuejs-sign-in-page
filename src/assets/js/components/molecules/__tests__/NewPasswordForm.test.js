import 'regenerator-runtime/runtime';
import { shallowMount } from '@vue/test-utils';
import NewPasswordForm from '../NewPasswordForm.vue';

describe('NewPasswordForm', () => {
	let wrapper;

	beforeAll(() => { wrapper = shallowMount(NewPasswordForm); });

	it('Matches snapshot', () => { expect(wrapper.element).toMatchSnapshot(); });
});
