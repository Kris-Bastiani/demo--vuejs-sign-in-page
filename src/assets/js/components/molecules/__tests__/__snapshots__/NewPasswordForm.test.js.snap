// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewPasswordForm Matches snapshot 1`] = `
<form
  title="new password form"
>
  <p
    class="message _success"
    style="display: none;"
  >
    Your password was successfully changed.
  </p>
  <input-field-stub
    label="New password:"
    placeholder="Enter new password"
    required="true"
    type="password"
    value=""
  />
  <input-field-stub
    class="mrg_b__lrg"
    label="Confirm password:"
    placeholder="Confirm new password"
    required="true"
    type="password"
    value=""
  />
  <submit-button-stub
    class=""
    disabled="true"
    label="Save"
  />
</form>
`;
